### DISCUSSION TITLE
Example sites using GoBlog?

### DISCUSSION THREAD

**Discussion by gingerbeardman** [jlelse/GoBlog#74] [Show and tell]
Are there a list of sites that are using GoBlog?

Using Google to search for the "Connect via Tor. What is Tor?" text at the bottom of the blog I've only been able to find a few:

- ~~https://hacdias.com~~
- https://theopjones.blog
- https://jlelse.blog
- https://jlelse.dev
- https://goblog.app
- more below

Is there a better way?

The task is made more difficult by there being many other projects that share the same name "goblog". 

GoBlog seems to be really cool, so I'm interested to see what different people have done with it. Thanks

**Comment by kandr3s** [comment]
Hey, I've been using GoBlog for a while now at https://kandr3s.co - Haven't made many changes as I like the default UI and all functionality I need is already built-in.

I think https://hacdias.com runs on [Eagle](https://github.com/hacdias/eagle) and GoBlog uses some of its libraries.

**Comment by gingerbeardman** [comment]
> Haven't made many changes as I like the default UI

This is really what I want to see.

**Comment by jlelse** [comment]
Hey, nice to see interest in GoBlog! 😄 

Because there are no metrics, no specific hidden HTML or anything, I just know of people who have been contacting me and so far that's only @kandr3s who's (still) using GoBlog.

I didn't know about Theo Jones' site for example.

Let's pin this issue and people can add there sites, if they want to share them.

**Comment by gingerbeardman** [comment]
I've updated the OP

**Comment by jlelse** [comment]
> I haven’t done really anything to GoBlog itself, but I am running it also. Here’s a “wish list” I put together of what I’m going to look into as I learn the system a bit more.

Thanks for this post!

The random post link is a bug and will be fixed ASAP. (Update: Should be fixed now.)

Regarding the 410: You can delete the short link from the database and also the path from the "deleted" table. But you're right, it needs a better solution here. A 410 is useful though, because when sending webmentions, it indicates, that the mentions should be deleted, because the page no longer exists. Maybe I can add an option or configuration to edit this behavior.

For the custom CSS: I opened #54 .

Regarding image handling: When using one of the compression options, images should be limited to max 2000 px width or height.

Good ideas with editor side-by-side with preview and slug generated from title, will think about a good solution there. 😄

**Comment by AmurgCodru** [comment]
I've just migrated  from hugo to goblog for https://andreiclinciu.net  - Will give it a go and see how I like it:)
